django/thunderstore/repository/comment.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+from thunderstore.community.models import PackageListing
+from thunderstore.repository.models import Comment
+
+
+def clean_content(content: str) -> str:
+    return content.strip()
+
+
+class CreateCommentForm(forms.ModelForm):
+    class Meta:
+        model = Comment
+        fields = ["content"]
+
+    def __init__(
+        self,
+        user: User,
+        commented_object: PackageListing,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.user = user
+        self.commented_object = commented_object
+
+    def clean_content(self) -> str:
```

Comment:
I am not completely sure if manually defining `clean_content` will prevent the normal validation from being run, or should you call it from the superclass. Can you verify? We probably want to ensure the normal validation is also called

Reply:
Tested in `test_create_comment_too_long` and `test_edit_comment_too_long`, the max length check still works. I have also added `test_clean_content` and switched to using unclean contents in `test_create_comment` and `test_edit_comment` to make sure *both* `clean_content` and the max length checks run